workflow-dev/tpl/generate_params.toml

```diff
@@ -25,8 +25,10 @@ pullPolicy = "Always"
 dockerTag = "canary"
 
 [s3]
-accesskey = "YOUR KEY HERE"
-secretkey = "YOUR SECRET HERE"
+#Your aws access key. Leave it empty if you are on AWS and want to use IAM credentials
```

Comment:
This could still be cleaned up a bit, with a leading space and a final period:
`# Your AWS access key. Leave it empty if you want to use IAM credentials.`
Same for below.